### PR TITLE
Fixes #3045 - Skip empty log arrays

### PR DESCRIPTION
**Pull request by GregSutcliffe**

### CODE REVIEW

**Comment by domcleal** [comment]
Could you add a test please?  JSON's in my bug report :)

**Comment by GregSutcliffe** [comment]
@domcleal done

**Comment by domcleal** [comment]
Thanks @GregSutcliffe, merged as 823e4ba.

File: test/unit/report_test.rb [hunk 1]
```diff
@@ -127,6 +127,11 @@ def setup_user operation
     assert r
   end
 
+  test "it should import reports where logs is nil" do
```
Comment (isratrade) [inline test/unit/report_test.rb:130]: @GregSutcliffe, should the word 'not' be added to the test title -- "should **not** import?

Also, as an FYI, another way to do this to convert nil to an array.  [].each wont execute the loop

``` ruby
Array.wrap(report['logs']).each do |r|
```

Comment (GregSutcliffe) [inline test/unit/report_test.rb:130]: no, the title is correct - the report should still import correctly when the logs are not present. We still have metrics and a reported_at to process.

The Array tip is nice, thanks.

**Pull request closed without merging**